pkg/webhook/certs.go

```diff
@@ -67,8 +67,9 @@ func AddRotator(mgr manager.Manager) (error) {
 	mgr.Add(rotator)
 
 	reconciler := &ReconcileVWH{
-		Client: mgr.GetClient(),
+		client: mgr.GetClient(),
 		scheme: mgr.GetScheme(),
+		ctx:    context.Background(),
```

Comment:
No need to make a change, but just commenting: to really make this code healthy WRT context, you would need to add a context as the first parameter of AddRotator(...) and so on all the way up to the cmd/ file that originates all of this code and call context.Background() there. However that is a much larger change, so...

Reply:
Ack, ya there is definitely a larger context cleanup to be done across the code base and maybe in core k8s libraries themselves. Calling this out-of-scope for now.